### ISSUE TITLE
No getBinaryStream option for blob

### ISSUE THREAD

**Issue by joellara445** [BoardiesITSolutions/Android-MySQL-Connector#7]
**Is your feature request related to a problem? Please describe.**
Yes, i cannot pull in image from mysql

**Comment by boardy** [comment]
I don't think it supports blob data types yet. Let me have a look and see if I can implement it for you.

**Comment by boardy** [comment]
Hi, Just to let you know I haven't forgotten about this, I'm still looking into it but having some trouble with it for some reason.

**Comment by boardy** [comment]
Hi, 

Really sorry for the delay in getting this resolved for you, got a bit tied up on something and this caused some issues I hadn't realised due to larger data response being returned from the server which took me a while to pin down where the problem was. 

However, I'm happy to say, this has now been resolved, so you can call getBlob(String columnName) from your MySQL row which will return you a byte[] array. 

Then to get your image from the byte array in to your image view you would be able to do something similar to the following:

```
byte[] content = row.getBlob("ImageContent");
ImageView image = findViewById(R.id.test_image);
Bitmap bmp = BitmapFactory.decodeByteArray(content, 0, content.length);
image.setImageBitmap(bmp);
```

This has been released under version 0.26 so just need to amend your gradle file with this version and resync. 

Let me know how you get on. 

Thanks

Chris
